docs/database-engine/configure-windows/server-memory-server-configuration-options.md

```diff
@@ -118,26 +118,29 @@ To enable the lock pages in memory option:
   
 ## Providing the maximum amount of memory to SQL Server  
 Memory can be configured up to the process virtual address space limit in all [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] editions. For more information, see [Memory Limits for Windows and Windows Server Releases](/windows/desktop/Memory/memory-limits-for-windows-releases#physical_memory_limits_windows_server_2016).
-  
-## Examples  
-  
-### Example A  
- The following example sets the `max server memory` option to 4 GB:  
-  
-```sql  
-sp_configure 'show advanced options', 1;  
-GO  
-RECONFIGURE;  
-GO  
-sp_configure 'max server memory', 4096;  
-GO  
-RECONFIGURE;  
-GO  
-```  
-  
+
+## Examples
+
+### Example A. Set the max server memory option to 4 GB.
+ The following example sets the `max server memory` option to 4 GB.  Note that although `'show advanced options'` specifies the name of the option as `max server memory (MB)`, the example demonstrates omitting the `(MB)`.

```

Comment:
Please change: 
> 'show advanced options' specifies 

to

> `sp_configure` displays

Reply:
@jzabroski - if you can make these changes, we will be able to update the article.

Thank you for your submission.

Reply:
@MikeRayMSFT  Sorry, FastHub on android doesnt seem to have a Resolve Conversation button. I fixed this last night.